playbook-website/app/javascript/site_styles/_opacity_guidelines.scss

```diff
@@ -0,0 +1,48 @@
+ul {
+   
+  &.opacity-play {
```

Comment:
Not sure I like this selector name. What are we styling here? Can we be more specific? `-play` seems too generic and doesn't tell me a lot about the context.

Reply:
We aren't styling anything, the way I built the opacity circles, is I used "pb--utlities-color", but it was giving it margins and borders so I used "opacity-play" to get rid of all of that. I am not sure what to name it ;u; '' Do you have a suggestion?